_assets/js/netlify/generateChildElements.js

```diff
@@ -0,0 +1,25 @@
+const generateChildElements = (html) =>
+  html.map((item) => {
+    if (item.nodeType === Node.TEXT_NODE) return item.wholeText;
+    // To Do: If it's not an element node, we prolly don't care about it, but we should check other types
+    // to make sure.
+    if (item.nodeType !== Node.ELEMENT_NODE) return;
+    const itemHasChildren = item.childNodes?.length > 0;
+    const content = [];
+    if (itemHasChildren) {
+      content.push(generateChildElements([...Array.from(item.childNodes)]));
+    }
+    const attrsObj = {};
+    if (item.attributes.length > 0) {
+      for (const key of Array.from(item.attributes)) {
+        attrsObj[key.name] = key.value;
+      }
+    }
```

Comment:
Another way of writing this:

```suggestion
    const attrs = Object.fromEntries([...item.attributes].map(
      attr => [attr.name, attr.value]
    ));
```

<img width="481" alt="image" src="https://user-images.githubusercontent.com/15126660/212180315-8ba54fbb-8a71-4794-aced-8f54890fa497.png">

Reply:
This is a good call. Much simpler and removes a variable. Thanks for catching!